lib/bokken_web/views/file_view.ex

```diff
@@ -0,0 +1,16 @@
+defmodule BokkenWeb.FileView do
+  use BokkenWeb, :view
+  alias BokkenWeb.FileView
+
+  def render("index.json", %{files: files}) do
+    %{data: render_many(files, FileView, "file.json")}
+  end
+
+  def render("show.json", %{file: file}) do
+    %{data: render_one(file, FileView, "file.json")}
+  end
+
+  def render("file.json", %{file: file}) do
+    %{id: file.id, document: file.document, description: file.description}
```

Comment:
`file.document` should be the path to the document, using `Document.url` function.